.NET/Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishDateTimePeriodParserConfiguration.cs

```diff
@@ -64,7 +65,7 @@ public EnglishDateTimePeriodParserConfiguration(ICommonDateTimeParserConfigurati
             PrefixDayRegex = EnglishDateTimePeriodExtractorConfiguration.PrefixDayRegex;
             BeforeRegex = EnglishDateTimePeriodExtractorConfiguration.BeforeRegex;
             AfterRegex = EnglishDateTimePeriodExtractorConfiguration.AfterRegex;
-
+            TasksmodeMealTimeofDay = EnglishDateTimePeriodExtractorConfiguration.TasksmodeMealTimeofDay;
```

Comment:
Nitpick: Mode this init to the last and add a blank line before it.

Reply:
changes integrated